### ISSUE TITLE
[BUG] Simple Calendar Error - read fix inside

### ISSUE THREAD

**Issue by Resfelm** [fantasycalendar/FoundryVTT-RestRecovery#114]
**Describe the bug**
If both Rest Recovery and Simple Calendar are enabled, clicking on the Long Rest button on any player character sheet produces an error and the Rest Recovery options window does not show up.

**To Reproduce**
1. Enable both Rest Recovery 5e and Simple Calendar in a game world
2. Open any player character sheet and click on the L.Rest button bellow the Hit Dice display.

**Expected behavior**
The Rest Recovery options window should pop up.

**Screenshots**
![image](https://user-images.githubusercontent.com/25343704/208401466-28d05855-fc42-41eb-8aca-ab573123af8f.png)

**Setup:**
 - Foundry Version: 10.291
 - System Version: DND5e 2.0.3
 - Rest Recovery Version: 1.4.3
 - Browser & version: Google Chrome 108.0.5359.125 (Official Build) (64-bit)

**Active modules:**
 - Rest Recovery
 - Simple Calendar
 - Many Others

**Additional context**
None

**Comment by Haxxer** [comment]
Please reinstall simple calendar, this is explained in Simple Calendar under the release notes for version 2.0.4:

> Update Issue Fix
> Right now Foundry says that version 1.3.75 is newer than version 2.0.1 and won't update the module. I will explain why below.
> 
> TL;DR: I made a mistake, the quick fix is to uninstall and reinstall Simple Calendar. There will be no data loss and the issue will
> be solved going forward.
> 
> Longer Version: When I first release Simple Calendar I did a silly thing and put a "v" in front of the version that Foundry reads. > Foundry uses this version to determine if an update is a newer version or not.
> 
> This version is also stored in the module's details so not something that can be fixed without updating the module.
> 
> When Foundry compares a version that starts with a v to one that does not start with a v, the one that starts with a v is > considered newer. As well the first portion of two versions that both start with a v, example 'v1' and 'v2', are considered to be the same. Until version 2 of Simple Calendar this didn't matter as it was always comparing 'v1' to 'v1'. FoundryVTT won't change how they check version so I need to make the change on my side.
> 
> Unfortunately the easiest way to fix this is to uninstall the previous version of Simple Calendar and reinstall it. After doing this, it will update properly going forward. When uninstalling a module that modules' data is not removed so there is no data loss in doing this. That said if you are concerned about data loss please make a backup of your world(s) before doing this exercise.
> 
> I apologize about this mistake and the extra work/confusion that may have come because of it.

I will leave this ticket open so that others can see it.

**Comment by Resfelm** [comment]
Thank you for the speedy reply, I did take a look to see if I could find anything related to this error but I was not thorough enough it seems.
